ansible/templates/hosts

```diff
@@ -0,0 +1,13 @@
+; the first parameter in a host is the inventory_hostname which has to be
```

Comment:
@rabbah I wanted to comment on the whole file. is this possible? didn't find a button for it so I took the first line.